src/api/v4/error-handling/carto-error.js

```diff
@@ -1,4 +1,20 @@
+var _ = require('underscore');
+var retrieveFriendlyError = require('./error-list');
 var UNEXPECTED_ERROR = 'unexpected error';
+var GENERIC_ORIGIN = 'generic';
+
```

Comment:
a small description would be nice here to understand this method without reading the code

Reply:
:+1: